test_suite.py

```diff
@@ -12,11 +12,15 @@
     except FileNotFoundError:
         print("Download https://drive.google.com/file/d/1kvFxYh2fo3bHfVj1OKKLX8YzuxswHvYg/view?usp=sharing and extract it to a directory named 'problems' in this folder.")
         sys.exit(1)
+
+    passes = fails = 0
     for prob in sorted(problems, key=lambda x: x.name):
         # Luck / weird problems
-        if prob.name in ["123", "123 Reloaded", "Error", "Helloworld", "Inverse Quine", "Not Quine", "Quine", "Timeout"]: continue
-        # Ruby version stuff
+        if prob.name in ["123", "123 Reloaded", "Error", "Inverse Quine", "Not Quine", "Palindromic Quine", "Quine", "Timeout"]: continue
+        # Ruby version / float / rational trickery
         if prob.name in ["area of triangle", "0_5 broken keyboard", "Cancel fractions", "Equal Temperament", "MIDI note number to frequency", "Numloop"]: continue
+        # Sisyphus's dump was unluckily case-insensitive, so these have wrong answers mixed in:
+        if prob.name.lower() in ["helloworld", "christmas tree", "multiplication table"]: continue
```

Comment:
Also on my system (which is case insensitive), "Tic Tac Toe" is broken due to this issue